### PR TITLE
Add/products

### PR DESCRIPTION
**Pull request by tanishk15sharma**
- Added products to `db`
- Refactored categories filter
- select category from home page

### CODE REVIEW

File: src/components/productListing/ProductsContainer.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ const ProductsContainer = () => {
       try {
         let response = await axios.get("/api/products");
         setProducts(response.data.products);
+        console.log(response);
```
Comment (Mohitkumar33) [inline src/components/productListing/ProductsContainer.js:25]: remove console.log

**Review by deepak29-git — APPROVED** [review]

File: src/components/productListing/ProductsContainer.js [hunk 2]
```diff
@@ -22,6 +22,7 @@ const ProductsContainer = () => {
...
         setLoad(false);
       } catch (err) {
         console.log(err);
```
Comment (Mohitkumar33) [inline src/components/productListing/ProductsContainer.js:28]: you can throw error console.errror

File: src/utilities/filters-utils.js [hunk 3]
```diff
@@ -12,28 +12,35 @@ const getFilteredProducts = (
...
-    .filter(
-      (product) =>
-        product.name.toLowerCase().includes(search.toLowerCase()) ||
-        product.brand.toLowerCase().includes(search.toLowerCase())
-    );
+  return (
+    products
+      .filter((product) => (stockBoolean ? true : product.inStock))
+      .filter((product) => (deliveryBoolean ? product.fastDelivery : true))
+      // .filter((product) => categoryObj[product.categories])
```
Comment (Mohitkumar33) [inline src/utilities/filters-utils.js:25]: remove commented code

**Review by Mohitkumar33 — APPROVED** [review]
You are doing good

File: src/components/productListing/ProductsContainer.js (hunk 1 above)
Comment (tanishk15sharma) [reply src/components/productListing/ProductsContainer.js:25]: sure

File: src/components/productListing/ProductsContainer.js (hunk 2 above)
Comment (tanishk15sharma) [reply src/components/productListing/ProductsContainer.js:28]: okay

File: src/utilities/filters-utils.js (hunk 3 above)
Comment (tanishk15sharma) [reply src/utilities/filters-utils.js:25]: sure

**Pull request merged**